packages/coin-support-utils/src/getAccountHistory/index.ts

```diff
@@ -52,6 +52,29 @@ async function getTransactions(
   );
 }
 
+function fillMissingPriceHistory(
+  history: IPriceSnapshot[],
+  days: 1 | 7 | 30 | 365,
+) {
+  const expectedEntries = days === 365 ? days : days * 24;
+  const startTime = history[0].timestamp;
+  const interval = days === 365 ? 24 * 60 * 60 * 1000 : 60 * 60 * 1000;
```

Comment:
This is also defined in line `123`. It's better if we have only 1 definition

Reply:
changed